examples/custom-theme.html

```diff
@@ -19,7 +19,7 @@
   var $ = document.querySelector.bind(document);
   window.onload = function () {
     Ps.initialize($('#Default'), {
-      theme: 'big-and-ugly'
+      theme: 'custom-theme'
```

Comment:
i noticed this example was not using an actual custom theme, so I added a custom theme that this example now loads.